src/components/PostDetails.tsx

```diff
@@ -1,22 +1,18 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { PostsContext } from './PostContext';
 
 export const PostDetails: React.FC = () => {
+  const { selectedPost } = useContext(PostsContext);
```

Comment:
currently it doesn't work:
![Screenshot from 2023-11-29 18-51-18](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/86923089/b683f4e0-4921-4c07-9c27-67558af171f5)

Please, complete the task